docs/posts/roadmap-2024-H1/index.qmd

```diff
@@ -0,0 +1,383 @@
+---
+title: "Ibis project 2024 roadmap"
+author: "Cody"
+date: "2024-02-05"
+thumbnail: commits.png
+categories:
+    - blog
+    - roadmap
+    - community
+---
+
+## Overview
+
+The Ibis project is an [independently governed open-source
+community](https://github.com/ibis-project/governance) that builds the portable
+Python dataframe library. It is primarily backed by [Voltron
+Data](https://voltrondata.com) but has
+[contributors](https://github.com/ibis-project/ibis/graphs/contributors) across
+a range of data companies and institutions.
+
+This is the first public roadmap for the Ibis project. Thus, there's a bit of
+backstory to explain how we got here and setup the context for where we're
+going. Feel free to [skip ahead to the roadmap](#2024-roadmap) if you're already
```

Comment:
Clicking on `skip ahead to the roadmap` doesn't jump to the roadmap for me. Is that just a me thing?

Reply:
Ah, the anchor link is just `#roadmap`, must be a quarto thing?

Reply:
![image](https://github.com/ibis-project/ibis/assets/417981/53b77005-65e0-4a34-bcf7-0d656dd1d56a)

Reply:
```suggestion
going. Feel free to [skip ahead to the roadmap](#roadmap) if you're already
```

Reply:
same...looking at the URL I guess I need to ignore the numbers, so just "#roadmap" here -- will change that, good catch